bindings/python/runtime.cpp

```diff
@@ -97,7 +101,5 @@ void bind_runtime(py::module_ &m) {
 }
 
 PYBIND11_MODULE(_kllvm_runtime, m) {
-  initStaticObjects();
-
```

Comment:
If I understand correctly, the root cause of all these memory leaks is that this is not the right place to call this initializer function, but instead it has to be called from the imported module in Python. Is there a simple explanation for why that is?

Reply:
I don't have a satisfactory explanation for that, unfortunately. I suspect the answer probably lies somewhere in the internals of how Pybind11 actually works 🤷

Reply:
(but yes, your understanding matches mine)